keep-alive/src/test/java/KeepAliveGattTest.kt

```diff
@@ -350,6 +354,148 @@ class KeepAliveGattTest {
         }
     }
 
+    @Test
+    fun `When 'connect' occurs, the onConnect lambda is called once`() = runBlocking {
+        val bluetoothDevice = mockBluetoothDevice()
+        val gatt1 = mockk<Gatt> {
+            every { onCharacteristicChanged } returns flow { delay(Long.MAX_VALUE) }
+            coEvery { disconnect() } returns Unit
+        }
+
+        mockkStatic(BLUETOOTH_DEVICE_CLASS) {
+            var onConnectLambdaCalls = 0
```

Comment:
```suggestion
            var onEventActionCalls = 0
```

Reply:
As I've changed the "onEventAction" terminology, I've changed this to "onConnectCalls"